scripts/eosio_build_darwin.sh

```diff
@@ -159,8 +159,11 @@
 						printf "\\tExiting now.\\n\\n"
 						exit 1;
 					fi
-                                        if [[ "$DEP" == "llvm@4" ]]; then
-                                                "${BREW}" unlink ${DEP}
+					DEP=${DEP//libusb /}
+					if [[ "$DEP" == "llvm@4" ]]; then
+						"${BREW}" unlink ${DEP}
+					elif [[ -z "$DEP" ]]; then
+						true
 					elif ! "${BREW}" unlink ${DEP} && "${BREW}" link --force ${DEP}
```

Comment:
This here is all sorts of WTFBBQ. The script seems written such that for any dependencies we're installing unlink them, _but if we fail to unlink force link them_. That just really boggles my mind. Also bizarre is that `DEP` appears to be a _list_ of items. So the `if [[ "$DEP" == "llvm@4" ]];` checks seems... not right. How could DEP ever be "llvm@4" when the line to change DEP is `DEP=$DEP"${brewname} "` -- it has a space after every DEP.

I need libusb to remain linked, which is its default, but that `unlink ${DEP}` burns me. So add some stuff here to remove libusb from the DEP list and skip the unlink/force link if DEP is then empty because of that removal.